<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectExercise</class>
 <widget class="QDialog" name="selectExercise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomePT와 함께 하는 올바른 홈트레이닝 라이프</string>
  </property>
  <property name="windowIcon">
   <iconset resource="heart.qrc">
    <normaloff>:/icon_heart/heart.png</normaloff>:/icon_heart/heart.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:&quot;Aliceblue&quot;;</string>
  </property>
  <widget class="QLabel" name="selectTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>650</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt&quot;경기천년제목M Medium&quot;; color:&quot;black&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;원하는 운동을 선택해주세요.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="exercise1" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>290</width>
     <height>172</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/exercise1/ex1.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="exercise2" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>290</width>
     <height>172</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/exercise2/ex2.jpg);
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exButton1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:&quot;Dodgerblue&quot;; color:&quot;white&quot;;font: 16pt&quot;경기천년제목M Medium&quot;;</string>
   </property>
   <property name="text">
    <string>플랭크</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exButton2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:&quot;Dodgerblue&quot;; color:&quot;white&quot;;font: 16pt&quot;경기천년제목M Medium&quot;;</string>
   </property>
   <property name="text">
    <string>스쿼트</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="heart.qrc"/>
  <include location="e1.qrc"/>
  <include location="e2.qrc"/>
 </resources>
 <connections/>
</ui>
